.github/workflows/test.yml

```diff
@@ -53,3 +53,17 @@ jobs:
         run: |
           forge test
         id: test
+
+      - name: Check committed FHE contract binding
+        run: |
+          forge bind \
+            --bindings-path ./crates/fhe-contract-tests/src/bindings \
+            --module --single-file \
+            --select "^FHE\$"
+
+      - name: Run FHE contract on anvil tests
+        run: |
+          cargo run --release --manifest-path=crates/fhe-contract-tests/Cargo.toml -- node &
```

Comment:
More typical usage would be `cargo run -p fhe-contract-tests`, but if this works that's fine

Reply:
Fixed.